lib/spack/docs/configuration.rst

```diff
@@ -66,25 +66,44 @@ are multiple configuration scopes. From lowest to highest precedence:
    here, but should override them in other configuration scopes. The
    defaults here will change from version to version of Spack.
 
+#. **site-admin**: Stored in ``$spack/etc/site-admin``, which does not
+   exist by default. This scope contains private settings shared by
+   those administering a Spack instance on behalf of other
+   users. These settings affect only *this instance* of Spack. This
+   scope is meant to have read access restricted to control which
+   users are affected by these settings. This scope only overrides
+   defaults.
+
 #. **system**: Stored in ``/etc/spack/``. These are settings for this
    machine or for all machines on which this file system is
-   mounted. The system scope can be used for settings idiosyncratic to a
-   particular machine, such as the locations of compilers or external
-   packages. These settings are presumably controlled by someone with
-   root access on the machine. They override the defaults scope.
+   mounted. The system scope can be used for settings idiosyncratic to
+   a particular machine, such as the locations of compilers or
+   external packages. These settings are presumably controlled by
+   someone with root access on the machine. They override the defaults
+   and site-admin scopes.
 
 #. **site**: Stored in ``$(prefix)/etc/spack/``. Settings here affect
-   only *this instance* of Spack, and they override the defaults and system
-   scopes. The site scope can be used for per-project settings (one
-   Spack instance per project) or for site-wide settings on a multi-user
-   machine (e.g., for a common Spack instance).
+   only *this instance* of Spack, and they override the defaults,
+   system, and site-admin scopes. The site scope can be used for
+   per-project settings (one Spack instance per project) or for
+   site-wide settings on a multi-user machine (e.g., for a common
+   Spack instance).
 
 #. **plugin**: Read from a Python package's entry points. Settings here affect
    all instances of Spack running with the same Python installation. This scope takes higher precedence than site, system, and default scopes.
 
-#. **user**: Stored in the home directory: ``~/.spack/``. These settings
-   affect all instances of Spack and take higher precedence than site,
-   system, plugin, or defaults scopes.
+#. **user**: Stored in the home directory:
+   ``~/.local/config/spack/``. These settings affect all instances of
+   Spack and take higher precedence than site, system, plugin, or
+   defaults scopes.
+
+#. **per-spack**: Stored in the home directory:
```

Comment:
It is currently called `this-spack`, I used to call it `per-spack-user`, but that was nixed. I'm open to other names (or can brainstorm it more)

Reply:
I prefer ``this-spack`` as well. Should be good (once I push).